asserts/pool.go

```diff
@@ -155,6 +168,26 @@ type unresolvedRec struct {
 	err error
 }
 
+func (u *unresolvedRec) isAssertionNewer(a Assertion) bool {
+	return a.Revision() > u.at.Revision
+}
+
+func (u *unresolvedRec) groupingPtr() *internal.Grouping {
+	return &u.grouping
+}
+
+func (u *unresolvedRec) label() Grouping {
+	return u.serializedLabel
+}
+
+func (u *unresolvedRec) isRevisionNotKnown() bool {
```

Comment:
thanks for the change,  isAssertionNewer and isRevisionNotKnown make a more coherent pair than also having full access to the revision